### PR TITLE
Add --css and --json options

### PR DESCRIPTION
**Pull request by pascalw**
~~(This PR depends on https://github.com/Wakamai-Fondue/wakamai-fondue-cli/pull/1. I'll rebase this PR when #1 is merged).~~

This PR adds two CLI options: `--css` and `--json`, to output CSS (for all features) and to dump a JSON object with font metadata.

```
Usage: wakamai-fondue [options] <file>

Options:
  -V, --version  output the version number
  -c, --css      Generate CSS
  -j, --json     Dump info about font in JSON format
  -h, --help     display help for command
```

### CODE REVIEW

**Comment by pascalw** [comment]
@RoelN I've picked `wakamai-fondue [options] <file>` for usage now. `[options]` and `<file>` can also be given in reverse order. So usage looks like this:

```sh
wakamai-fondue /path/to/font.ttf --css
wakamai-fondue /path/to/font.ttf --info
```

We could also work with subcommands, which would look like this:

```sh
wakamai-fondue css /path/to/font.ttf
wakamai-fondue info /path/to/font.ttf
```

IMO the former is more suitable, as the font will always be a required argument anyway and what you want to get out of the tool can easily be specified with a flag (`--css`, `--info`).

**Comment by pascalw** [comment]
I also left out writing results to a file, this can easily be done using pipes:

```sh
wakamai-fondue /path/to/font.ttf --css > font.css
wakamai-fondue /path/to/font.ttf --json > font.json
```

File: index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+const { stdout } = require("process");
+
+program
+	.name("wakamai-fondue")
+	.version(packageJson.version)
+	.arguments("<file>")
+	.action(run);
+
+program.option("-c, --css", "Generate CSS");
+program.option("-i, --info", "Dump info about font");
```
Comment (pascalw) [inline index.js]: I'm not too sure about the name. Perhaps this should be called `--json`. As in: `What can my font do? Tell me in JSON please.`.

Comment (RoelN) [reply index.js]: Good suggestion, with `--info` I would assume some basic font info, not "everything". Go for it! 👍

**Comment by RoelN** [comment]
Flags are `--fine`, and piping the output to a file yourself is fine too! 👍  This works super nicely, thanks!

**Review by RoelN — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (pascalw) [reply index.js]: Renamed to `--json`.

**Comment by pascalw** [comment]
@RoelN I renamed `--info` to `--json` and optimized a few things for NPM distribution (binary named `wakamai-fondue`, don't include tests etc in published package).

**Comment by RoelN** [comment]
Nice improvements! All looking good 👍

**Pull request merged**